hack/ci/fetch-chart-dependencies.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### This script fetches dependencies for a chart, based on the lock file
+### contained within the charts' directory. The script iterates over all charts.
+
+set -euo pipefail
+
+BASEDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../.."
+REALDIR="$(cd "$(dirname $(readlink -f "${BASH_SOURCE[0]}"))" && pwd)"
+source ${REALDIR}/../lib.sh
+
+cd ${BASEDIR}
+charts=$(find charts/ -name Chart.yaml | sort)
+
+[ -n "$charts" ] && while read -r chartYAML; do
+  dirname="$(dirname $(echo "$chartYAML"))"
+  name="$(yq read "$chartYAML" name)"
+  chartname=$(yq read "$chartYAML" name)
```

Comment:
which yq are you using?
Mine gives me this:

```shell
$ yq read charts/backup/velero/Chart.yaml name
Error: unknown command "read" for "yq"
Run 'yq --help' for usage.

$ yq --version
yq (https://github.com/mikefarah/yq/) version 4.14.2
```

Reply:
I'm using the same yq that's used in CI - I also had to change mine locally :/
```
> yq --version
yq version 3.4.1
```

Reply:
yeah...

```bash
$ docker run -it --rm quay.io/kubermatic/build:go-1.17-node-14-1 yq --version
yq version 3.4.1
```

outrages!

Reply:
cc @xrstf